tests/commands/DriveToWaypointCommandTest.cpp

```diff
@@ -0,0 +1,96 @@
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <cstdio>
+
+#include "../../src/commands/CommandBase.h"
+#include "../../src/commands/DriveToWaypointCommand.h"
+
+using namespace robot::types;
+using namespace std::chrono_literals;
+
+void assertApprox(double expectedThetaVel, double expectedXVel,
+                  const commands::command_t& actual, double angle = 1e-5,
+                  double vel = 1e-5) {
+  bool velEqual = abs(expectedThetaVel - actual.thetaVel) <= angle;
+  bool angleEqual = abs(expectedXVel - actual.xVel) <= vel;
+
+  if (velEqual && angleEqual) {
+    SUCCEED();
+  } else {
+    printf(
+        "Expected (thetaVel,xVel)=(%.5f,%.5f), "
+        "Actual (thetaVel,xVel)=(%.5f,%.5f)\n",
+        expectedThetaVel, expectedXVel, actual.thetaVel, actual.xVel);
+    FAIL();
+  }
+}
+
+const double ATAN_4_3 = 0.927295218002; // value of atan(4/3) in rad.
+const double THETA_KP = 3.5;
+const double NORMAL_DRIVE_SPEED = 4.0;
+const double SLOW_DRIVE_SPEED = 2.0;
+const double DONE_THRESHOLD = 0.25;
+const double CLOSE_TO_TARGET_DUR_VAL = 1.0;
```

Comment:
Can you put all of these (and the additional helper method above) in an anonymous namespace? They should be private to this file.

See: https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces